### PR TITLE
Company in reference cases

### PR DESCRIPTION
**Pull request by Semperdecus**

### CODE REVIEW

File: src/components/uielements/CasePreview.astro [hunk 1]
```diff
@@ -58,21 +58,23 @@ const { caseData, small } = Astro.props;
...
+          caseData?.company?.logo?.dark?.asset._id && (
             <SanityImg
               loading="lazy"
-              src={caseData?.metadata?.image}
+              src={caseData?.company?.logo?.dark?.asset?._id}
               class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
               height={60}
-              alt={caseData?.metadata?.image?.asset?.altTexts?.[locale] || ""}
+              alt={
+                caseData?.company?.logo?.dark?.asset?.altTexts?.[locale] || ""
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:76]: did you check if the alt data is filled? I saw you did ... in the query. But I'm not sure if that query also get the altexts data

File: src/components/uielements/CasePreview.astro [hunk 2]
```diff
@@ -58,21 +58,23 @@ const { caseData, small } = Astro.props;
...
+            set:html={caseData?.company?.name}
           />
           <h2
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
             set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData?.metadata?.image?.asset._id && (
+          caseData?.company?.logo?.dark?.asset._id && (
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:69]: I would still add a ? after asset here. asset?.id

**Review by DavithkbY — CHANGES_REQUESTED** [review]

File: src/components/uielements/CasePreview.astro (hunk 1 above)
Comment (Semperdecus) [reply src/components/uielements/CasePreview.astro:76]: yes altdata is filled:
image
image

**Comment by DavithkbY** [comment]
lgtm

**Review by DavithkbY — APPROVED** [review]

**Pull request merged**